README.md

```diff
@@ -31,7 +31,7 @@ Options
 | digestAlgorithm     | digest algorithm, options: sha1, sha256          | ```'sha256'``` |
 | RelayState          | state of the auth process                        | ```req.query.RelayState || req.body.RelayState``` |
 | sessionIndex          | the index of a particular session between the principal identified by the subject and the authenticating authority                        | _SessionIndex is not included_ |
-| responseHandler       | custom response handler for SAML response f(SAMLResponse, req, res, next) | HTML response that POSTS to postUrl |
+| responseHandler       | custom response handler for SAML response f(SAMLResponse, req, res, next, postUrl, opts) | HTML response that POSTS to postUrl |
```

Comment:
Maybe `f(SAMLResponse, postUrl, opts, req, res, next)` ?

please also update [this one](https://github.com/vandernorth/node-samlp/blob/master/lib/samlp.js#L204)